Modules/syslogmodule.c

```diff
@@ -61,10 +61,22 @@ module syslog
 
 #include "clinic/syslogmodule.c.h"
 
-/*  only one instance, only one syslog, so globals should be ok  */
-static PyObject *S_ident_o = NULL;                      /*  identifier, held by openlog()  */
+/*  only one instance, only one syslog, so globals should be ok,
+ *  these fields are writable from the main interpreter only. */
+static PyObject *S_ident_o = NULL;  // identifier, held by openlog()
 static char S_log_open = 0;
 
+static inline int
+is_main_interpreter()
```

Comment:
```suggestion
is_main_interpreter(void)
```

Otherwise, the function can be called with any number of arguments.